File: GroupA-Cust-Acc-Trans-Api (4)/AccountsApiService/Models/Account.cs
namespace AccountsApiService.Models
{
    public class Accounts
    {
        public long AccountID { get; set; }
        public decimal Balance { get; set; }
        public bool hasCheque  { get; set; }
        public int wd_quota   { get; set; }
        public int dp_quota   { get; set; }
        public bool isActive  { get; set; }
        public int CustomerID  { get; set; }
        public int type_id   { get; set; }
        public string BranchID  { get; set; }
    }
}

File: GroupA-FrontEnd-Angular/src/app/models/account.ts
export class Account {
  constructor(
    public accountID: number,
    public balance: number,
    public hasCheque: boolean,
    public wd_quota: number,
    public dp_quota: number,
    public isActive: boolean,
    public customerID: number,
    public type_id: number,
    public branchID: string
  ) {}
}
